bitcoinplusplus/layer-2/lightning-on-taproot.md

```diff
@@ -1,70 +1,70 @@
 And Alice's signature is a commitment to the nonce that she used, her public key, and the message.
 In order to make sure that the signature doesn't tweak her private key, we tweak the hash of her commitment with the message multiplied with her private key by a little random value called r.
 And we tweak that because if we weren't to tweak that, then somebody would trivially be able to apply the modular multiplicative inverse to lowercase s here and extract Alice's private key, which, of course, we see to avoid.
 But even doing everything correctly here, there are some risks with opening, with creating two of two signatures, for example.
 One of those risks is naive key aggregation.
 The other one that I'm gonna talk about later is not so used, but what do I mean by naive key aggregation?
-So let's say that we have Alice and Bob, who are trying to open a channel between the two of them.
+So, let's say that we have Alice and Bob, who are trying to open a channel between the two of them.
 And specifically, they're trying to figure out what their shared public key is gonna be.
 Obviously, somebody has gotta start in that protocol.
-So let's say that Alice initiates the conversation and she sends her public key to Bob, and then Bob is supposed to send his public key to Alice, and the sum of those two pubkeys is going to be the pubkey that's going to be the shared public key for the store signature.
-Well, Bob can quite trivially do an attack where having received Alice's public key first, he first generates his own pubkey, but then instead of sending his random pubkey back to Alice, he subtracts Alice's pubkey.
-And then as a result, we have that the sum of the two pubkeys that they calculate is one that completely eliminates Alice's.
-So It ends up that Bob has the capability to unilaterally create a signature.
+So, let's say that Alice initiates the conversation, and she sends her public key to Bob, and then Bob is supposed to send his public key to Alice, and the sum of those two pub keys is going to be the pub key that's going to be the shared public key for the store signature.
+Well, Bob can quite trivially do an attack where having received Alice's public key first, he first generates his own pub key, but then instead of sending his random pub key back to Alice, he subtracts Alice's pub key.
+And then as a result, we have that the sum of the two pub keys that they calculate is one that completely eliminates Alice's.
+So, It ends up that Bob has the capability to unilaterally create a signature.
 And of course, you do not want lightning channels where one of the parties is able to just unilaterally close it and unilaterally update the state without the other parties buying.
 So that is one of the issues that require some solution.
 Another issue, of course, is nonce reuse.
 So here I have a quite simple equation where we have two separate signatures for different messages, m' and m'', but we're using the same nonce.
```

Comment:
As pointed out in the Review Guidelines, you should ensure that code-related technical terms or math equations are enclosed in backticks (`) to enhance clarity. 

There are a few other places that you should do that in this transcript 
```suggestion
So here I have a quite simple equation where we have two separate signatures for different messages, `m'` and `m''`, but we're using the same nonce.
```